techmatters/flex-plugins
# Add Support to view more languages in pdf, including Thai

Primary reviewer: @stephenhand 

## Description
- Replace 'Open Sans' with Roboto for better character font support and use NotoSansThaiLooped for Thai characters for viewing pdf documents (for `react-pdf/renderer` library)
- The reasoning for using NotoSansThaiLooped was because Noto supports languages and fonts used worldwide: 
See https://fonts.google.com/noto/specimen/Noto+Sans and https://notofonts.github.io/
 * In the future, when more languages are added, adding a compatible fontFamily might be necessary. Check https://notofonts.github.io/

### Checklist
- [n/a] New tests added
- [n/a] Feature flags added
- [n/a] Strings are localized
- [n/a] Tested for chat contacts
- [n/a] Tested for call contacts

### Related Issues
Fixes # [1618](https://tech-matters.atlassian.net/browse/CHI-1618)

### Verification steps
- In `HrmFormPlugin.tsx`, change line 71 to 
`definitionVersion: DefinitionVersionId.huV1,` to check hungarian (Ukrainian character are supported as well now)
`definitionVersion: DefinitionVersionId.thV1,` to check thai character support
- Insert some phrases of those languages in `components/case/casePrint/CasePrintView.tsx` or its child components or hardcode values that will appear in the PDF format
<img width="784" alt="Screenshot 2023-01-05 at 6 53 58 PM" src="https://user-images.githubusercontent.com/102122005/210902616-b5657dbf-e68a-407a-a8a9-3b29729b505f.png">
